compiler/src/iree/compiler/Codegen/Utils/GPUUtils.cpp

```diff
@@ -22,6 +23,63 @@ static constexpr unsigned kShuffleBitWidth = 32;
 namespace mlir {
 namespace iree_compiler {
 
+//===----------------------------------------------------------------------===//
+// Target architecture utils
+//===----------------------------------------------------------------------===//
+
+static constexpr StringLiteral kCudaTarget = "cuda";
```

Comment:
THis seems very CUDA specific. Can we move this into `LLVMGPU/`?

Reply:
I moved that into kernelConfig in LLVMGPU